app/controllers/hourglass/api_base_controller.rb

```diff
@@ -139,6 +140,10 @@ def internal_server_error(e)
       respond_with_error :internal_server_error, Rails.env.production? ? t('hourglass.api.errors.internal_server_error') : messages, no_halt: true
     end
 
+    def already_booked(_e)
+      respond_with_error :bad_request, t('hourglass.api.time_logs.errors.already_booked'), no_halt: true
```

Comment:
Is this a new translation? Do we have to add it?

Reply:
It already exists. But we could add a special translation for the overlapping time tracker/log stopping error.